utilities/openemr-cmd/openemr-cmd

```diff
@@ -496,190 +554,191 @@ case "$FIRST_ARG" in
+		run_devtools_in_docker "${CONTAINER_ID}" sql-ssl-off
 		;;
 	ssc)
-		$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE "$DEV_TOOLS sql-ssl-client"
+		run_devtools_in_docker "${CONTAINER_ID}" sql-ssl-client
 		;;
 	ssco)
-		$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE "$DEV_TOOLS sql-ssl-client-off"
+		run_devtools_in_docker "${CONTAINER_ID}" sql-ssl-client-off
 		;;
 	css)
-		$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE "$DEV_TOOLS couchdb-ssl"
+		run_devtools_in_docker "${CONTAINER_ID}" couchdb-ssl
 		;;
 	cso)
-		$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE "$DEV_TOOLS couchdb-ssl-off"
+		run_devtools_in_docker "${CONTAINER_ID}" couchdb-ssl-off
 		;;
 	csc)
-		$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE "$DEV_TOOLS couchdb-ssl-client"
+		run_devtools_in_docker "${CONTAINER_ID}" couchdb-ssl-client
 		;;
 	csco)
-		$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE "$DEV_TOOLS couchdb-ssl-client-off"
+		run_devtools_in_docker "${CONTAINER_ID}" couchdb-ssl-client-off
 		;;
 	lss)
-		$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE "$DEV_TOOLS ldap-ssl"
+		run_devtools_in_docker "${CONTAINER_ID}" ldap-ssl
 		;;
 	lso)
-		$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE "$DEV_TOOLS ldap-ssl-off"
+		run_devtools_in_docker "${CONTAINER_ID}" ldap-ssl-off
 		;;
 	lsc)
-		$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE "$DEV_TOOLS ldap-ssl-client"
+		run_devtools_in_docker "${CONTAINER_ID}" ldap-ssl-client
 		;;
 	lsco)
-		$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE "$DEV_TOOLS ldap-ssl-client-off"
+		run_devtools_in_docker "${CONTAINER_ID}" ldap-ssl-client-off
 		;;
 	gmb|generate-multisite-bank)
-		$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE "$DEV_TOOLS generate-multisite-bank $@"
+        # Pass remaining arguments ($@) correctly quoted to the devtools function
+		run_devtools_in_docker "${CONTAINER_ID}" generate-multisite-bank "$@"
 		;;
 	el)
-		$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE "$DEV_TOOLS enable-ldap"
+		run_devtools_in_docker "${CONTAINER_ID}" enable-ldap
 		;;
 	dld)
-		$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE "$DEV_TOOLS disable-ldap"
+		run_devtools_in_docker "${CONTAINER_ID}" disable-ldap
 		;;
 	xl)
-		$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE "$DEV_TOOLS xdebug-log"
+		run_devtools_in_docker "${CONTAINER_ID}" xdebug-log
 		;;
 	lxp)
-		$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE "$DEV_TOOLS list-xdebug-profiles"
+		run_devtools_in_docker "${CONTAINER_ID}" list-xdebug-profiles
 		;;
 	bad)
-		$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE "$DEV_TOOLS build-api-docs"
+		run_devtools_in_docker "${CONTAINER_ID}" build-api-docs
 		;;
 	lm)
-		$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE "$DEV_TOOLS list-multisites"
+		run_devtools_in_docker "${CONTAINER_ID}" list-multisites
 		;;
-
 	*)
-		$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE "$DEV_TOOLS $FIRST_ARG"
+		# Default case: assume it's a devtools command
+		run_devtools_in_docker "${CONTAINER_ID}" "${FIRST_ARG}" "$@"
 		;;
 esac
-exit $FINAL_EXIT_CODE
+exit "${FINAL_EXIT_CODE}"
```

Comment:
With `set -e` it's hard to be sure the script will exit with exactly this code unless we do a lot more review.

Reply:
I agree, but `set -e` was in place already and I simply added the unbound var and pipefail bits. I think it's probably OK for the moment.